# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325878462 18000
#      Fri Jan 06 14:34:22 2012 -0500
# Branch stable
# Node ID 4f04284f45e93ab2169408aa37dd39ae3b1199ca
# Parent  a737b3fb9c4d89d3694da6b4e623aeee64b212e1
Added tag ss-3-5-92 for changeset a737b3fb9c4d

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -50,8 +50,9 @@ 6c69a7c39039bccd5a2fe481fcc7f896c82b160b
 3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9 ss-3-3-53
 bd2643f0ce57d070963bedd48857405f6924aa85 ss-3-3-54
 695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
 901d466ee55ac902a875ec0ade6f1eccef0841dc release-3-4-1
 3666e8e6f96e6899b8306d6ea9614aadf0500d67 release-3-4-2
 b0e70a71647b671ebcfa7a79af1ae6d3c0f52065 release-3-4-3
 3781981be535e80d44c85373b8fdaa60ca5cd097 ss-3-5-90
 ff5588774680d4f54567311fc109c8e351950f1c ss-3-5-91
+a737b3fb9c4d89d3694da6b4e623aeee64b212e1 ss-3-5-92
